base/pcre2/stone.yml

```diff
@@ -0,0 +1,39 @@
+name        : pcre2
+version     : "10.40"
+release     : 1
+summary     : Perl-Compatible Regular Expressions
+license     : BSD-3-Clause
+homepage    : https://github.com/PCRE2Project/pcre2
+description : |
+    The PCRE2 library is a set of C functions that implement regular expression pattern matching using the same syntax and semantics as Perl 5
+upstreams   :
+    - https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.40/pcre2-10.40.tar.gz: ded42661cab30ada2e72ebff9e725e745b4b16ce831993635136f2ef86177724
+builddeps   :
+    - make
+    - bzip2-devel
+    - pkgconfig(zlib)
+    # Check deps:
```

Comment:
`checkdeps` are a thing ;)

Reply:
![](https://c.tenor.com/dJGc_g65qu4AAAAC/huzzah-frazzled.gif)

Reply:
When are checkdeps installed? During the initial package install during the build or are they deferred until just before the check step? I imagine the former because the latter doesn't make any sense.